src/indexing.jl

```diff
@@ -25,7 +58,7 @@ end
 
 @inline function getindex(m::AbstractMultiScaleArray, i, I...)
     if isempty(m.values) || i < length(m.end_idxs)
-        length(I) == 1 ? m.nodes[i].nodes[I[1]] : m.nodes[i][I...]
```

Comment:
I'm not sure what this line does? Seems like it can error in a couple of ways... The tuple version is not type stable currently

Reply:
It handles the case where `I` is a scalar since then it cannot splat I think.